tests/test_graphql_api.py

```diff
@@ -27,6 +27,13 @@ def test_post_json(client: TestClient):
     assert response.json() == {"data": {"hello": "Hello stranger"}}
 
 
+@pytest.mark.filterwarnings("ignore::DeprecationWarning")
```

Comment:
Which the `DeprecationWarning` are we ignoring here? I don't think we really want to silence those anyway, do we? :)

Reply:
The warning that is ignored is raised from the invalid escape character in the test.

Reply:
Can you share what the warning is exactly? Maybe that's something that needs fixing.

Reply:
Here is the warning. The travis build fails because 
`DeprecationWarning: invalid escape sequence \{ "/", data="\{test}", headers={"content-type": "application/json"}`

Reply:
Ignored that warning in flake8 as well. Those warnings are ignored because the invalid escape sequence is the invalid JSON to be passed in to trigger the 400 error.

Reply:
Okay. :) Can we send another kind of invalid JSON that doesn't trigger this warning? For example, a unterminated object:

```python
response = client.get("/", data="{ 'test':", headers={"content-type": "application/json"})
```
`client.post("/", json="{ test:"`

Reply:
Ok changed to unterminated object :)